test/libethereum/blockchainHelper.cpp

```diff
@@ -0,0 +1,261 @@
+/*
```

Comment:
there is blockchain.cpp and Blockchain.cpp already.
in this file I tried out helping blockchain-friendly classes for working with core blockchain class and testing it.
so I called it helper. Ideally I want to transfer the overwhelming complex logic of test file blockchain.cpp to this testing classes. It is just so complex right now  (but working fine) that I don't want to do refactoring so not to break anything.